mm2src/coins/eth.rs

```diff
@@ -524,6 +525,26 @@ impl EthCoinImpl {
     }
 }
 
+async fn get_raw_transaction_impl(coin: EthCoin, req: RawTransactionRequest) -> RawTransactionResult {
+    if req.tx_hash.len() < 2 {
+        return MmError::err(RawTransactionError::InvalidHashError(req.tx_hash));
+    }
+    let hash = H256::from_str(&req.tx_hash[2..req.tx_hash.len()])
```

Comment:
Let it also work if the hash doesn't have `0x` prefix. Like here: 
https://github.com/KomodoPlatform/atomicDEX-API/blob/454c571bc493d48930822f41122b5e590593c7f7/mm2src/coins/eth.rs#L1109

Reply:
it's done.